scripts/pipe_reader_writer.sh

```diff
@@ -1,25 +1,29 @@
-#!/bin/bash -x
+#!/bin/bash
 
 # Create a pipe to read
 # Delete if already exists
-rm osmosis_command_reader.fifo 2>/dev/null
-mkfifo osmosis_command_reader.fifo
+rm /tmp/osmosis_command/command_reader.fifo 2>/dev/null
+mkfifo /tmp/osmosis_command/command_reader.fifo
+
+rm /tmp/osmosis_command/result_reader.fifo 2>/dev/null
+mkfifo /tmp/osmosis_command/result_reader.fifo
 
 
 while true
 do
     while read cmd
     do
-        $cmd 2> /tmp/command_reader_error;
-        # Write to another named pipe 'osmosis_result_reader' which is created by python app
+        $cmd 2> /tmp/osmosis_command/command_reader_error;
+        # Write to another named pipe '/tmp/osmosis_command/result_reader.fifo' which is created by python app
         # NOTE: the following outputs multiline errors into single, which is desired
         # at the moment
-	errcode=$?
-	if [[ $errcode -ne 0 ]]; then
-           errmsg=`cat /tmp/command_reader_error`
+        errcode=$?
+        if [[ $errcode -ne 0 ]]; then
+            errmsg=`cat /tmp/osmosis_command/command_reader_error`
         else
-	   errmsg="no errrors reported"
+            errmsg="no errrors reported"
         fi
-        echo "$? $errmsg" >> osmosis_result_reader.fifo
-    done < osmosis_command_reader.fifo
+        echo "$? $errmsg" >> /tmp/osmosis_command/result_reader.fifo
+    done < /tmp/osmosis_command/command_reader.fifo
+    sleep 1
 done
```

Comment:
This script runs in host. so `/tmp` is host location here. However in the python code it refers to path in the container. This is because, this script is run in host.

Reply:
By script, you are referring to **osmosis**, right?

This is a bit tricky as there is no mention of osmosis in this repo and the setup of osmosis has to be done by the user themselves. We could update docker-compose.yml to make the `/tmp/osmosis_comman` directory available to the host, making the host osmosis tool reachable. I am doing the same in [posm-build](https://github.com/posm/posm-build/pull/132/files#diff-423deb13b7c401b1a7f41ee91c77f722e11d2f317d6a66b546524e8a04cc8b03R114). Sounds good?

Reply:
By script I am referring to the file itself 'pipe_reader_writer.sh`.
Yes, sharing `/tmp/osmosis_command` in docker_compose.yml will work perfect. It's just that, it should be included in this PR as well right?

Reply:
Updated docker-compose.yml https://github.com/posm/posm-replay-server/pull/23/commits/e3632c11cd082b94f8ff6c4be0e7b83999baacde